docs/release-source/release/fake-timers.md

```diff
@@ -108,31 +112,64 @@ setTimeout(function () {
 }, 35);
 ```
 
-Please refer to the `lolex.install` [documentation](https://github.com/sinonjs/lolex#var-clock--lolexinstallconfig) for the full set of features available and more elaborate explanations.
+Using fake timers with `async` / `await`:
 
-*Since `sinon@3.0.0`*
+```
+async function asyncFn() {
 
-`var clock = sinon.useFakeTimers([now, ]prop1, prop2, ...)` is no longer supported. To define which methods to fake, please use `config.toFake`.
+    await wait(100);
+
+    console.log('resolved 1', Date.now());
+
+    await wait(10);
+
+    console.log('resolved 2', Date.now());
+}
+
+async function test() {
+
+    const clock = sinon.useFakeTimers();
+
+    setTimeout(() => console.log('timeout', Date.now()), 200);
+
+    asyncFn(); // NOTE: no `await` here - it would hang, as the clock is stopped
+
+    await clock.tickAsync(200);
+}
+    
+// test() prints: 
+// - resolved 1 100
+// - resolved 2 110
+// - timeout 200
+```
+
+Note that in the above example, the synchronous `clock.tick(200)` would only print `timeout 200` and `resolved 1 200`. 
 
 
-#### `clock.tick(time);`
+#### `clock.tick(time);` / `await clock.tickAsync(time)`
 
 Tick the clock ahead `time` milliseconds.
 
 Causes all timers scheduled within the affected time range to be called. `time` may be the number of milliseconds to advance the clock by or a human-readable string. Valid string formats are "08" for eight seconds, "01:00" for one minute and "02:34:10" for two hours, 34 minutes and ten seconds.
 
-time may be negative, which causes the clock to change but won't fire any callbacks.
+`time` may be negative, which causes the clock to change but won't fire any callbacks.
```

Comment:
You can actually remove this whole line. I [removed support for negative ticks](https://github.com/sinonjs/lolex/blob/master/CHANGELOG.md#300--2018-10-08) in 3.0